src/index.html

```diff
@@ -1,17 +1,179 @@
+          <label for="tems-and-conditions">I accept the term of the agreement</label>
+          <input
+      type="checkbox"
+      name="checkbox"
+      id="tems-and-conditions"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="new-field">
+        <legend>
+        Registration
+        </legend>
+        <div class="input-class">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            autocomplete="off"
+            required
+          >
+        </div>
+        <div class="input-class">
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            name="password"
+            minlength="8"
+            maxlength="30"
+            id="password"
+            autocomplete="off"
+            required
+          >
+        </div>
+      </fieldset>
+      <fieldset class="new-field">
+        <legend>
+        An interesting fact about you!
+        </legend>
+        <div class="input-class">
+          <label for="animals">Do you love cats?
+            <input
+      type="radio"
+      name="yes"
+      id="animals"
+            >
+        Yes
+          </label>
+          <label>
+            <input
+      type="radio"
+      name="yes"
+            >
+        No
+          </label>
+        </div>
+        <div class="input-class">
+          <label for="color">What is your favorite color?</label>
+          <input
+      type="color"
+      name="color"
+      id="color"
+          >
+        </div>
+        <div class="input-class">
+          <label for="bed-time">What time do you go to bed?</label>
+          <input
+      type="time"
+      name="bed-time"
+      id="bed-time"
+          >
+        </div>
+        <div class="input-class">
+          <label for="car">What are your favorite brand of cars
+            <select name="car" multiple>
+              <option value="bmw">BMW</option>
+              <option value="toyota">Audi</option>
+              <option value="audi">Lada</option>
+            </select>
+          </label>
+        </div>
+        <div class="input-class">
+          <label for="range">How do you rate our work?</label>
+          <input
+      type="range"
+      name="range"
+      id="range"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="new-field">
+        <legend>
+          Additional info
+        </legend>
+        <div class="input-class">
+          <label for="comments">Comments:
+            <textarea name="textarea"></textarea>
+          </label>
+        </div>
+        <div class="input-class">
+          <label for="recommendations">Would you recommend us?</label>
+          <select name="recommendations">
+            <option value="yes"selected>yes</option>
+            <option value="no">no</option>
+          </select>
+        </div>
```

Comment:
```suggestion
        <div class="input-class">
          <label for="comments">
            Comments:
            <textarea name="textarea"></textarea>
          </label>
        </div>
        
        <div class="input-class">
          <label for="recommendations">Would you recommend us?</label>
          
          <select name="recommendations">
            <option value="yes" selected>yes</option>
            <option value="no">no</option>
          </select>
        </div>
```
add blank line between multiline elements everywhere

Reply:
Done! Thank you!